src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -0,0 +1,297 @@
+        public void FixedStatementMultipleDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int i1, i2;
+    static void Main(string[] args)
+    {
+        var reference = new Program();
+        unsafe
+        {
+            fixed (/*<bind>*/int* p1 = &reference.i1, p2 = &reference.i2/*</bind>*/)
+            {
+
+            }
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IFixedStatement (OperationKind.FixedStatement)
+  IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: System.Int32* p1 (OperationKind.VariableDeclaration)
+      Initializer: IAddressOfExpression (OperationKind.AddressOfExpression, Type: System.Int32*)
+          IFieldReferenceExpression: System.Int32 Program.i1 (OperationKind.FieldReferenceExpression, Type: System.Int32)
+            Instance Receiver: ILocalReferenceExpression: reference (OperationKind.LocalReferenceExpression, Type: Program)
+    IVariableDeclaration: System.Int32* p2 (OperationKind.VariableDeclaration)
+      Initializer: IAddressOfExpression (OperationKind.AddressOfExpression, Type: System.Int32*)
+          IFieldReferenceExpression: System.Int32 Program.i2 (OperationKind.FieldReferenceExpression, Type: System.Int32)
+            Instance Receiver: ILocalReferenceExpression: reference (OperationKind.LocalReferenceExpression, Type: Program)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+";
+            VerifyOperationTreeForTest<VariableDeclarationSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void UsingStatementDeclaration()
+        {
+            string source = @"
+using System;
+
+class Program : IDisposable
+{
+    static void Main(string[] args)
+    {
+        using (/*<bind>*/Program p1 = new Program()/*</bind>*/)
+        {
+        }
+    }
+
+    public void Dispose() {}
+}
+";
+            string expectedOperationTree = @"
+IUsingStatement (OperationKind.UsingStatement)
+  IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: Program p1 (OperationKind.VariableDeclaration)
+      Initializer: IObjectCreationExpression (Constructor: Program..ctor()) (OperationKind.ObjectCreationExpression, Type: Program)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+";
+            VerifyOperationTreeForTest<VariableDeclarationSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void UsingStatementMultipleDeclarations()
+        {
+            string source = @"
+using System;
+
+class Program : IDisposable
+{
+    static void Main(string[] args)
+    {
+        using (/*<bind>*/Program p1 = new Program(), p2 = new Program()/*</bind>*/)
```

Comment:
>new Program() [](start = 38, length = 13)

Consider also testing with the value missing and with equals value missing (two different scenarios).